### ISSUE TITLE
Time attribute contains element tags in JUnit report

### ISSUE THREAD

**Issue by LogMANOriginal** [JKISoftware/Caraya#178]
Using Caraya 1.4.1, `time` attribute values for test suites and test cases in JUnit reports contain element tags, which breaks parsers.

Actual
```xml
<testsuite time="<time>0.123</time>">
<testcase time="<time>0.123</time>">
```

Expected
```xml
<testsuite time="0.123">
<testcase time="0.123">
```

Here is a test case:
![Time attribute should not contain element tags](https://github.com/JKISoftware/Caraya/assets/5776685/8567af47-dd2d-4c58-a3a8-9862620af439)

**Comment by francois-normandin** [comment]
Confirmed.

Since XML is not forcing dot-notation, but can support locale-specific numbers. I made your test compatible with locale settings for both comma and dot separated values. (Note that the use can set the desired locale strategy in the report class)
![image](https://github.com/JKISoftware/Caraya/assets/11728548/87b8a3fc-d931-48ca-bb18-b752cc022db5)

The following snippet (set as value) has been inserted in the Report.JUnit routine where time was concerned.
![image](https://github.com/JKISoftware/Caraya/assets/11728548/739db162-b983-4fa7-b1a4-434255ba7df0)
